db/migrate/20170207173837_set_service_display_and_retired_to_false.rb

```diff
@@ -0,0 +1,12 @@
+class SetServiceDisplayAndRetiredToFalse < ActiveRecord::Migration[5.0]
+  Service = Class.new(ActiveRecord::Base)
```

Comment:
There is always "deleting everything under app/ and then restoring after migrations" option :trollface: joking/not joking